k2/python/tests/intersect_dense_pruned_test.py

```diff
@@ -197,7 +221,7 @@ def test_two_fsas_long_pruned(self):
                                   device=device,
                                   requires_grad=True)
 
-            supervision_segments = torch.tensor([[0, 0, 95], [1, 20, 50]],
+            supervision_segments = torch.tensor([[0, 1, 95], [1, 20, 50]],
```

Comment:
The supervision segment has a start offset. So 0 is relative to the supervision segment, not to nnet_output. Do we need to add the start offset to `frame_idx`?